### ISSUE TITLE
Problem setting includes (Atom 1.0.0)

### ISSUE THREAD

**Issue by frepond** [AtomLinter/linter-erlc#5]
Hi, I'm having problems setting includes in Atom 1.0.0 (linter-erlc@0.3.0). The includes are not configured even if I use relative or absolute path to the include dir.

![screen shot 2015-06-30 at 1 33 56 pm](https://cloud.githubusercontent.com/assets/467290/8436183/b3f734e8-1f2c-11e5-823d-6a8929bbe7c6.png)

Regards

**Comment by Arcanemagus** [comment]
Marking as closed since this linter is in the process of being re-written. If you are still seeing the issue once #8 lands please feel free to file a new issue about it, and I'm sorry nobody responded till now!
